src/test/java/integration/ClientControllerIT.java

```diff
@@ -356,9 +358,9 @@ public void updateClientAsCompany() throws Exception {
 
         // check updated data
         Client updatedClient = clientService.getEntityByUuid(clientUuid, user);
-        long timeModified = updatedClient.getLastModified().getTime();
+        LocalDateTime timeModified = updatedClient.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).